src/api/objects/ObjectAPI.js

```diff
@@ -391,15 +393,14 @@ export default class ObjectAPI {
                 lastPersistedTime = domainObject.persisted;
                 const persistedTime = Date.now();
                 this.#mutate(domainObject, 'persisted', persistedTime);
-
                 savedObjectPromise = provider.update(domainObject);
             }
 
             if (savedObjectPromise) {
                 savedObjectPromise.then(response => {
                     savedResolve(response);
                 }).catch((error) => {
-                    if (lastPersistedTime !== undefined) {
+                    if (isNewObject) {
```

Comment:
Previously, this was checking if `lastPersistedTime` was defined, meaning that the object is not new. This seems to have the opposite check, and means `lastPersistedTime` can potentially be undefined. I think this should be `if (!isNewObject)` instead

Reply:
Oh dang... yep. Whoopsie!